apps/frontend/src/store/user-context.tsx

```diff
@@ -4,6 +4,7 @@ import { getStoredUser } from '../utils/auth';
 const UserContext = createContext({
   username: '',
   token: '',
+  id: ''
```

Comment:
I need to do some studying on this. Still not sure why we need it

Reply:
At that moment, we thought that the best way to have access to the user's information anywhere within the app was to use React context or Redux. It turned out that for the complexity of our app it is enough to retrieve user info directly from local storage.

Reply:
so, do we still need to use this, or are we leaving it there just in case?

Reply:
I would leave it just in case for now.